app/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
       'androidx.constraintlayout:constraintlayout:1.1.3',
       'androidx.core:core-ktx:1.0.2',
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
+      'androidx.recyclerview:recyclerview:1.0.0',
       'com.google.dagger:dagger:2.24',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
       "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1",
```

Comment:
use single quotes

Reply:
single quotes not yet updated

Reply:
Actually, I mentioned this in my last PR (#197 ), I am only adding a new recyclerview and I prefer not to update someone else's PR. The double quotes has been mentioned from Ben's PR.

Later on we can solve such nit issues by creating a separate PR.

Reply:
Okay